inductiva/simulators/openfast.py

```diff
@@ -0,0 +1,38 @@
+"""OpenFAST module of the API for wind turbine simulations."""
+from typing import Optional
+
+from inductiva import types, tasks, simulators
+
+
+class OpenFAST(simulators.Simulator):
+    """Class to invoke a generic OpenFAST simulation on the API.
+
+    """
+
+    def __init__(self):
+
+        super().__init__()
+        self.api_method_name = "openfast.openfast.run_simulation"
+
+    def run(self,
+            input_dir: types.Path,
+            commands: types.Commands,
+            on: Optional[types.ComputationalResources] = None,
+            storage_dir: Optional[types.Path] = "",
+            extra_metadata: Optional[dict] = None,
+            **kwargs) -> tasks.Task:
```

Comment:
How can I use the OpenMP here? Can we select the number of threads?

Reply:
As we can see in [this video](https://youtu.be/HS0KijUu-Uc?list=PLOkLYSdoXbp0x4fI19bxtFitpX9xQsNyw&t=275), in Windows we just need to call Fast.Farm (with openmp) and it will run in parallel depending on the number of turbines defined.

<img width="579" alt="Captura de ecrã 2024-04-03, às 10 48 23" src="https://github.com/inductiva/inductiva/assets/157368229/024cee1c-5a93-41a4-bf2b-785ab6731c74">

I could not find a working example to run FAST.Farm, so more tests are needed in the future.